packages/strapi-plugin-settings-manager/admin/src/containers/Home/sagas.js

```diff
@@ -45,7 +93,7 @@ export function* fetchLanguages() {
 
     const [appLanguagesData, listLanguagesData] = yield [
       call(request, requestUrlappLanguages, opts),
```

Comment:
I'm not a linter but the variable should follow the camelCase pattern and should be called `requestUrlAppLanguages`